src/_components/shared/highlight_banner.erb

```diff
@@ -0,0 +1,17 @@
+<highlight-banner>
+  <div class="wrapper">
+    <div class="content">
+      <h3><%= @title %></h3>
+      <%= content %>
+      <% if @link_url.present? %>
+        <%= link_to @link_url, class: "button" do %>
+          <%= svg "#{@link_icon}" if @link_icon.present? %>
+          <%= @link_label if @link_label.present? %>
```

Comment:
Do we need a condition to ensure that both `@link_icon` and `@link_label` are not blank?

Reply:
You're right, done!